domain/domain.go

```diff
@@ -358,10 +363,29 @@ func (do *Domain) Reload() error {
 
 	// Lock here for only once at the same time.
 	do.m.Lock()
-	defer do.m.Unlock()
+	defer func() {
+		do.lastReloadAt = time.Now()
```

Comment:
It's better to set it to `startTime`.

Reply:
I think lastReloadAt is more meaningful. We are tracing the duration between 2 reload operations, not one reload operation take how many seconds.

Reply:
Maybe `lastReloadedAt` is better?